hoomd/update/remove_drift.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+"""Implement RemoveDrift."""
+
+import hoomd
+from hoomd.operation import Updater
+from hoomd.data.typeconverter import NDArrayValidator
+from hoomd import _hoomd
+import numpy as np
+
+
+class RemoveDrift(Updater):
+    r"""Remove the average drift from a system restrained on a lattice.
+
+    Args:
+        reference_positions ((*N_particles*, 3) `numpy.ndarray` of
+            ``numpy.float64``): the reference positions of the
```

Comment:
```suggestion
            `float`): the reference positions of the
```
@b-butler is recommending using `float` for numpy array data types that store double precision floating point values.